packages/tasks/src/model-libraries.ts

```diff
@@ -518,6 +518,20 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	hallo: {
```

Comment:
It would be nice to have a least 1 model tagged as `"hallo"` before merging this PR (currently [none is tagged](https://huggingface.co/models?other=hallo)). To do this, you need to add `library_name: hallo` in the [model card metadata](https://huggingface.co/fudan-generative-ai/hallo/blob/main/README.md?code=true#L2)